### PR TITLE
Update and consolidate video sections

### PR DESCRIPTION
**Pull request by srpearce**
## [DOC-3643](https://openedx.atlassian.net/browse/DOC-3643)

Update and consolidate video sections pending the addition of an option to choose thumbnail images (for partners only).

### Date Needed (optional)

5 July 2017

### Reviewers

Possible roles follow. The PR submitter checks the boxes after each reviewer finishes and gives :+1:. 

- [ ] Subject matter expert: @muzaffaryousaf (**video/upload_video_edx/video_uploads.rst lines 86-96 only**)
- [ ] Doc team review (copy edit): @edx/doc
- [ ] Product review: @sstack22 

FYI: Tag anyone else who might be interested in this PR here.

### Testing

- [ ] Ran ./run_tests.sh without warnings or errors

### HTML Version (optional)

*Make sure to review both versions, because they're different.*

- [ ] **Partners**
  [Working with Video Components](http://draft-partner-july-2017-video-changes.readthedocs.io/en/latest/course_components/create_video.html) (single file)
  [Creating and Adding Video Content](http://draft-partner-july-2017-video-changes.readthedocs.io/en/latest/video/index.html) (whole section)

- [ ] **Open edX**
  [Working with Video Components](http://draft-open-edx-july-2017-video-changes.readthedocs.io/en/latest/course_components/create_video.html) (single file)
  [Creating and Adding Video Content](http://draft-open-edx-july-2017-video-changes.readthedocs.io/en/latest/video/index.html) (whole section)

### Post-review

- [ ] Add a comment with the description of this change or link this PR to the next release notes task.
- [ ] Squash commits

### CODE REVIEW

File: en_us/course_authors/source/video/upload_video_edx/video_uploads.rst [hunk 1]
```diff
@@ -165,13 +83,25 @@ partner support team. See :ref:`Create YouTube Channels`.
    A rectangular tile appears on the page for each file. The file name, a
    progress bar, and the status of the file upload process appear in the tile.
```
Comment (srpearce) [inline en_us/course_authors/source/video/upload_video_edx/video_uploads.rst]: @muzaffaryousaf Can you review lines 86-96 below? You don't need to review this entire PR.

File: en_us/course_authors/source/video/upload_video_edx/video_overview.rst [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+
+.. contents::
+  :local:
+  :depth: 1
+
+************************************
+Course Team Video Upload Overview
+************************************
+
+First, the edX partner support team works with the appointed video
```
Comment (debchatigny) [inline en_us/course_authors/source/video/upload_video_edx/video_overview.rst]: Should this be initial capitals (edX Partner Support Team)?

File: en_us/course_authors/source/video/upload_video_edx/video_overview.rst [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+administrators from your organization to establish accounts for your
+organization at YouTube™ and Amazon Web Services™ (AWS). For more information
```
Comment (debchatigny) [inline en_us/course_authors/source/video/upload_video_edx/video_overview.rst]: Suggest adding Amazon Web Services (AWS) to the glossary.

File: en_us/course_authors/source/video/upload_video_edx/video_setup.rst [hunk 4]
```diff
@@ -1,11 +1,17 @@
...
 ###########################
 
+.. note::
+  **This process applies only to courses that run on the edx.org site.** For
+  information about adding video files to courses that run on Edge, see
+  :ref:`Step 3. Upload a Video for an edx Edge Course <Post the Video Online>`.
+
 Before your course teams can begin to upload videos in Studio, you work with
-edX partner support to make sure that these preparatory tasks are complete.
+edX partner support to make sure that the following preparatory tasks are
```
Comment (debchatigny) [inline en_us/course_authors/source/video/upload_video_edx/video_setup.rst]: Initial capitalize edX Partner Support.

File: en_us/course_authors/source/video/upload_video_edx/video_uploads.rst [hunk 5]
```diff
@@ -150,9 +68,9 @@ parties identify and track video files over time.
 Upload Video Files
 ***************************
 
-Before you can upload video files, the video upload feature must be enabled
-for the course. Your video administrator coordinates this task with the edX
-partner support team. See :ref:`Create YouTube Channels`.
+Before you can upload video files, the video upload feature must be enabled for
+the course. Your video administrator coordinates this task with the edX partner
```
Comment (debchatigny) [inline en_us/course_authors/source/video/upload_video_edx/video_uploads.rst]: Initial capitalize edX Partner Support.

File: en_us/shared/video/additional_transcript_options.rst [hunk 6]
```diff
@@ -0,0 +1,109 @@
...
+Before you add a transcript file, make sure that each file has a unique name.
+If you use the same transcript name more than once, only the most recently
+added transcript will appear for every video component that is set up to use a
+transcript with that name. To avoid this problem, a best practice is to name
+additional transcript files by appending the transcript's language to the video
+file name.
+
+For example, you have two videos, named video1.mp4 and video2.mp4, and their
+original transcripts are named video1.srt and video2.srt. When you add Spanish
+transcripts for these videos, you name them video1_ES.srt and video2_ES.srt.
```
Comment (debchatigny) [inline en_us/shared/video/additional_transcript_options.rst]: As we recommend course creators use the ISO 639-1 codes in 9.8.2.3, should the locales be listed in lowercase (which is how I've typically seen them in file names)?

File: en_us/shared/video/create_transcript.rst [hunk 7]
```diff
@@ -0,0 +1,70 @@
+.. _Create the Transcript:
+
+#############################################
+Step 2. Create or Obtain a Video Transcript
+#############################################
+
+Transcripts are required for your videos. Transcripts help learners with
```
Comment (debchatigny) [inline en_us/shared/video/create_transcript.rst:7]: You could mention here that transcripts help all learners by providing an alternative way to consume the content, if they can't have sound on when watching videos, for example.

File: en_us/shared/video/create_video.rst [hunk 8]
```diff
@@ -0,0 +1,125 @@
...
+  Naming Conventions>` and :ref:`video compression specifications <Video
+  Compression Specifications>`.
+* The edX video player supports videos in .mp4, .mpeg, .webm, and .ogg format.
+  However, to help make sure all standard browsers can play your video, edX
+  **strongly** recommends that you use the .mp4 format.
+
+  .. only:: Partners
+
+    .. note::
+      For courses on edx.org, in a later step, you upload your videos to edX,
```
Comment (debchatigny) [inline en_us/shared/video/create_video.rst:39]: I'd suggest rewording: For courses on edx.org, in a later step, once you upload your videos to edX, edX automatically converts each of your video files into ..

File: en_us/shared/video/create_video.rst [hunk 9]
```diff
@@ -0,0 +1,125 @@
...
+   * - Max VBR
+     - 7.5 mbps
+     - 1.5 mbps
+   * - Audio
+     - AAC 44.1 / 192 kbps
+     - AAC 44.1 / 192 kbps
+
+.. note::
+ Typically you export at the same frame rate that was used when you created the
+ media file. For example, if you create the file in a country that uses the PAL
```
Comment (debchatigny) [inline en_us/shared/video/create_video.rst]: Do we have a file that captures various acronyms/video-specific terminology? If not, I'd suggest adding terms like VBR, PAL, fps, and NTSC to the glossary.

File: en_us/shared/video/create_video.rst [hunk 10]
```diff
@@ -0,0 +1,125 @@
...
+     - 1 mbps
+   * - Max VBR
+     - 7.5 mbps
+     - 1.5 mbps
+   * - Audio
+     - AAC 44.1 / 192 kbps
+     - AAC 44.1 / 192 kbps
+
+.. note::
+ Typically you export at the same frame rate that was used when you created the
```
Comment (debchatigny) [inline en_us/shared/video/create_video.rst]: Insert comma: Typically, you..

File: en_us/shared/video/upload_video.rst [hunk 11]
```diff
@@ -0,0 +1,39 @@
+Making course videos available to as many learners as possible can be a complex
+undertaking.
+
+* Videos need to be available from more than one host site to play in locations
+  around the world.
+* Videos need to be available in several different formats to play on
+  desktop computers, laptops, smartphones, and other mobile devices.
+* Videos need to be available for both download and streaming for learners with
+  slow or intermittent Internet connections.
```
Comment (debchatigny) [inline en_us/shared/video/upload_video.rst]: Internet is lowercase.

**Comment by muzaffaryousaf** [comment]
@mushtaqak can you review the related part here for video images/thumbnails ?(video/upload_video_edx/video_uploads.rst lines 86-96 only)

File: en_us/course_authors/source/video/upload_video_edx/video_uploads.rst [hunk 12]
```diff
@@ -165,13 +83,25 @@ partner support team. See :ref:`Create YouTube Channels`.
    A rectangular tile appears on the page for each file. The file name, a
    progress bar, and the status of the file upload process appear in the tile.
 
+#. (optional) Select a thumbnail image for the video.
+
+   The thumbnail image is the image that learners see before the video begins
+   to play. Studio provides a thumbnail image by default for your videos.
```
Comment (mushtaqak) [inline en_us/course_authors/source/video/upload_video_edx/video_uploads.rst]: @srpearce We are not showing default generated image to end user right now in this feature release. So we would probably remove this line `Studio provides a thumbnail image by default for your videos.`

So for now, we would 1st upload image through **Add Thumbnail** then if we want to update/change the image, we can use **Edit Thumbnail**

**Review by mushtaqak — CHANGES_REQUESTED** [review]
@srpearce I left a commit regarding video images. We are not yet showing showing default images for videos to user. Need to update it.

File: en_us/course_authors/source/video/create_preroll_video.rst [hunk 13]
```diff
@@ -1,13 +1,14 @@
 .. _Adding a PreRoll Video:
 
-*******************************************
-Adding a Pre-Roll Video to Your edX Course
-*******************************************
+################################################
+Add a Pre-Roll Video to Your edX Course
```
Comment (catong) [inline en_us/course_authors/source/video/create_preroll_video.rst:4]: @srpearce What's the reason for renaming the heading? Our standard naming conventions would have "Adding" for conceptual topics and "Add..." for the procedural topic.

File: en_us/course_authors/source/video/additional_transcript_options.rst [hunk 14]
```diff
@@ -0,0 +1,33 @@
...
+
+***************************************
+Upload an .sjson File (Deprecated)
+***************************************
+
+If your course uses .sjson files, you upload the .sjson file for the video
+to the **Files & Uploads** page, and then specify the name of the .sjson file
+in the video component.
+
+.. note::
```
Comment (catong) [inline en_us/course_authors/source/video/additional_transcript_options.rst]: Suggestion: move the note earlier, immediately following the heading?

File: en_us/course_authors/source/video/upload_video_edx/index.rst [hunk 15]
```diff
@@ -0,0 +1,37 @@
+.. _Upload a Video on edX:
+
+##############################################
+Step 3. Upload a Video for an edx.org Course
+##############################################
+
+.. note::
+  **This process applies only to courses that run on the edx.org site.** For
+  information about adding video files to courses that run on Edge, see
+  :ref:`Step 3. Upload a Video for an edx Edge Course <Post the Video Online>`.
```
Comment (catong) [inline en_us/course_authors/source/video/upload_video_edx/index.rst]: Is "Post the Video Online" the correct anchor? Looks like it should be "Upload Video"?

File: en_us/course_authors/source/video/upload_video_edx/index.rst [hunk 16]
```diff
@@ -0,0 +1,37 @@
...
+
+Making course videos available to as many learners as possible can be a complex
+undertaking.
+
+* Videos need to be available from more than one host site to play in locations
+  around the world.
+* Videos need to be available in several different formats to play on
+  desktop computers, laptops, smartphones, and other mobile devices.
+* Videos need to be available for both download and streaming for learners with
+  slow or intermittent Internet connections.
```
Comment (catong) [inline en_us/course_authors/source/video/upload_video_edx/index.rst]: lc "i" for "internet"

File: en_us/course_authors/source/video/upload_video_edx/video_overview.rst [hunk 17]
```diff
@@ -0,0 +1,116 @@
+.. _Video Processing Overview:
+
+###########################
+Video Processing Overview
+###########################
+
+.. note::
+  **This process applies only to courses that run on the edx.org site.** For
+  information about adding video files to courses that run on Edge, see
+  :ref:`Step 3. Upload a Video for an edx Edge Course <Post the Video Online>`.
```
Comment (catong) [inline en_us/course_authors/source/video/upload_video_edx/video_overview.rst]: Is "Post the Video Online" the correct anchor label?

File: en_us/course_authors/source/video/upload_video_edx/video_overview.rst [hunk 18]
```diff
@@ -0,0 +1,116 @@
...
+Uploads** page as the videos go through automated processing.
+
+.. note::
+ The edX automation process does not include captioning services. Your course
+ teams use your institution's current workflow to generate transcripts for your
+ video files, and then add them to the video in Studio. For more information,
+ see :ref:`Add a Video Transcript`.
+
+************************************
+Course Team Add Video Overview
```
Comment (catong) [inline en_us/course_authors/source/video/upload_video_edx/video_overview.rst]: This heading reads oddly and would be difficult to translate. Something like "Overview of Course Team Video Upload Process"?

File: en_us/course_authors/source/video/upload_video_edx/video_setup.rst [hunk 19]
```diff
@@ -124,9 +131,9 @@ more.
 Creating YouTube Channels
 ===============================
 
-.. note:: This procedure only needs to be completed once per course, but it
- must be complete before the course team begins to add videos to the course in
- Studio.
+.. note::
+ This procedure only needs to be completed once per course, but it must be
+ complete before the course team begins to add videos to the course in Studio.
```
Comment (catong) [inline en_us/course_authors/source/video/upload_video_edx/video_setup.rst]: "it must be complete" -> "it must be completed"

File: en_us/course_authors/source/video/upload_video_edx/video_uploads.rst [hunk 20]
```diff
@@ -184,9 +114,9 @@ page or by :ref:`downloading a report<Reporting Video Status>`.
 
 .. _Delete Videos from Upload Page:
 
-==================================================
-Remove Videos from the Video Uploads Page
-==================================================
+***************************
+Remove Video Files
```
Comment (catong) [inline en_us/course_authors/source/video/upload_video_edx/video_uploads.rst]: @srpearce Adding "Files" to the heading makes the actual impact of this procedure confusing - it's what I deliberately avoided when I worked on the story that enabled this cleanup.
@muzaffaryousaf can confirm, but files are not removed; this procedure just cleans up the listing of files on the Video Uploads page.

**Comment by catong** [comment]
@srpearce I left comments inline, and have a question about some flowcharts. In encoding_proc_overview.png. I'm confused about the "Upload" step having an arrow back to the "Assign Video ID" box, but I see the same thing in encoding_process_overview.png. However, the other flowcharts show the process ending after the upload step.

File: en_us/course_authors/source/video/create_preroll_video.rst (hunk 13 above)
Comment (srpearce) [reply en_us/course_authors/source/video/create_preroll_video.rst:4]: I'd like to maintain consistency with the other headings at that level:

screenshot 12

File: en_us/course_authors/source/video/upload_video_edx/video_uploads.rst (hunk 20 above)
Comment (srpearce) [reply en_us/course_authors/source/video/upload_video_edx/video_uploads.rst]: I probably wanted it to be consistent with the other heading ("Upload Video Files"), and I didn't know the files aren't actually deleted. I'll change this back.

**Comment by srpearce** [comment]
@catong The encoding_process.png flowchart is meant to show in more detail what happens in the encoding and hosting process (the overview flowchart just says "encode files" and "upload...", while the encoding_process.png flowchart shows the process encoding the original file into four separate files that are uploaded to four separate places). I'll try to make the overview flowchart more clear, however. (Hopefully the difference between the two is more clear when they're viewed in context: http://draft-partner-july-2017-video-changes.readthedocs.io/en/latest/video/upload_video_edx/video_overview.html.)

File: en_us/course_authors/source/video/upload_video_edx/video_uploads.rst (hunk 1 above)
Comment (srpearce) [reply en_us/course_authors/source/video/upload_video_edx/video_uploads.rst]: @mushtaqak Made the changes you requested; are lines 86-89 below now accurate?

Comment (mushtaqak) [reply en_us/course_authors/source/video/upload_video_edx/video_uploads.rst]: This is looking good 👍 

I have one suggestion/question  : Should we also tell user about **Edit Thumbnail** option if he/she wishes to change image ?

**Review by mushtaqak — APPROVED** [review]
@srpearce This is looking good 👍 Thanks.

I have left one suggestion/question regarding **Edit Thumbnail** if we want to tell user about it or not. It's up to you if you want to include it in docs.

File: en_us/course_authors/source/video/upload_video_edx/video_uploads.rst (hunk 1 above)
Comment (srpearce) [reply en_us/course_authors/source/video/upload_video_edx/video_uploads.rst]: @mushtaqak Yes, that's a great suggestion. Thanks!
